### ISSUE TITLE
Downloads stop before completed (95-97%).

### ISSUE THREAD

**Issue by wolfland** [inbasic/turbo-download-manager#50]
For some reason my downloads seem to stop before they are completed usually around
95-97% complete. If you pause the download and start it again it will complete.

It appears that the individual threads may be thinking the download is complete (see
screenshots) but the overall download is not. In other words the individual threads
have reached the end of the download but the larger overall download bar has not
showing that the overall download is not complete.

This started with a newer version of turbo download manager but I cannot pinpoint
the date when this started I just know that I did not have the problem until somewhat
recently.

OS is Ubuntu Linux 16.04 64bit , Firefox is version 46.0.1.

Thanks,
Brett

![screenshot from 2016-05-11 20-14-55](https://cloud.githubusercontent.com/assets/8964224/15451643/909bdb7c-1f9c-11e6-855d-c02671ab0496.png)
![screenshot from 2016-05-21 20-30-40](https://cloud.githubusercontent.com/assets/8964224/15451644/95cb7580-1f9c-11e6-8a84-352b4ad2bb1b.png)
![screenshot from 2016-05-21 21-29-50](https://cloud.githubusercontent.com/assets/8964224/15451645/99f8b82a-1f9c-11e6-89ce-135d9a989b06.png)

**Comment by inbasic** [comment]
Is this on 0.2.8? Can you please open the error console (Ctrl + Shift + J) while downloading a file and see if you get new errors? how about the log window ("i" button)?

**Comment by wolfland** [comment]
For some reason I was on 0.2.7. I have updated to 0.2.8 and will do some 
more downloads today and see if I still get the same failure. If so I 
will run the error console and log window and see if I can't collect 
some useful data for debugging.

Thank,
Brett

**Comment by wolfland** [comment]
Progress from this weekends tests:

First thing I did was upgrade to 0.2.8. I then tried to download a .iso 
from Sourceforge.net using
the iweb mirror. It failed at 98% and then at 99%. I captured the 
Firefox Console and InfoWindow
data but I did not have all data types running in the Firefox Console 
and the InfoWindow gave no
failure info.

So I tried again with another .iso and iweb again. This failed at 97% 
and I was able to capture more
info from the Firefox Console and well as pertinent info from the 
InfoWindow which indicated:

fetch error[3]; "timeout"

So I tried again with the same .iso and iweb and got the same failure:

fetch error[3]; "timeout"

So I switched mirrors to HeaNet and tried again. Both tries passed 
without an error.

So I switched mirrors back to iweb and tried again same file but with 
DownThemAll and it
passed without any problems.

So I tried Turbo again with iweb and same failure:

fetch error[3]; "timeout"

I think this is a legitimate timeout failure. Why it does not happen 
with DownThemAll I don't know.

I had to suspend testing due to a large thunderstorm but had one more 
thing I wanted to try.

The last thing I wanted to do was to repeat the testing on a machine 
that was connected directly
to my ethernet switch instead of running wireless just to be sure it 
wasn't some issue with wireless.

I ran the tests this afternoon and was able to download the same files 
using turbo without any
issues when connected directly to my ethernet switch.

So I am at a loss to explain the failures. I can make them occur at will 
when using turbo on my
wireless (N) setup and the iweb mirror but cannot get them to happen 
with wireless (N) and the
Heanet mirror or with DownThemAll.

Normally I would be using the Superb Internet 2 mirror but it is not 
always in the pool and so I
have had to use the iweb mirror as of late. Since turbo seems to work 
fine with Heanet and Superb
as well as with direct connection to my ethernet switch it would appear 
that Turbo is working just
fine. It may be some weird problem with either my wireless or the iweb 
mirror or both as when I
have seen timeouts in the past it has always been a legit problem with 
the server/mirror or the
internet path to the server/mirror(e.g. thunderstorm or server going 
down etc).

One of the ways I have seen timeouts reported is to change to overall 
progress bar to a different
color (I think DownThemAll uses blue) and then report the failure mode 
right on the progress bar
in text. This helps the user know that a failure has occurred as well as 
what the failure mode is.

Anyway I will close the bug report since the problem seems to be on my 
end and I will keep
working on it until I figure out what it is(probably a wireless conflict 
between devices or something).

Just in case you want to try and replicate it you go to Sourceforge.net 
and select
System Administration. Then select OS Distribution. You then select a 
project from
the list and once you enter the project page select files and look for a 
large .iso file
to download (e.g. 1Gb+). Select the file and when you are taken to the 
download page you
will see 2 entries in blue that you can select to either start the 
download manually or select
a particular mirror. Select mirror and if iweb is present in the list 
select it and the download
should start using the iweb mirror.  I suspect it will work fine for you.

Thanks for the debugging info it has been a big help and I will use it 
in the future before I bug
you again and thanks for the great download manager. Since the developer 
of DownThemAll
seems to have stopped any further development since Mozilla's 
announcement that they will
be changing how addons are handled/developed(last I checked anyway) 
Turbo has been the
best Download Manager choice under active development. Keep up the great 
work and I look
forward to seeing added features (love the download complete sound) as 
the product progresses
to release status.

Let me know if you want any of the data as I will keep it around just in 
case you want it.

Sorry for the false alarm,
Brett

**Comment by wolfland** [comment]
Closing issue. Problem was due to legitimate timeout failures not related to the Download Manager.
Thanks,
Brett

**Comment by inbasic** [comment]
Actually this is a bug. If for any reason a thread times out, TDM restarts this thread. I am not really sure why starting-trigger is not being emitted for you. I have done some changes to hopefully fix this issue.
Would you please try the latest dev version and let me know if the issue is fixed or not.

https://github.com/inbasic/turbo-download-manager/releases

You need to download the XPI one and drop it into your Firefox. If Firefox does not allow you to install an unsigned extension, please open about:debugging page in a new browser tab and load the downloaded XPI.

**Comment by wolfland** [comment]
Ok.. I have installed the 0.2.9 beta and will give it a try. Previously 
I could always get the iweb mirror
to fail every time so it should either pass or fail.

I am still eliminating variables at my end and have turned off all other 
wireless equipment that can
be disabled(e.g. keyboard/mice etc) as well as trying different USB 
ports and wireless hardware.

So far I have not been able to make any progress other than to note that 
other sourceforge mirrors
don't seem to have an issue other than iweb.

I will do more downloads tonight using the iweb mirror and let you know 
what happens.

Thanks for all the hard work,
Brett

**Comment by inbasic** [comment]
> I am still eliminating variables at my end

There is no need. The whole idea behind TDM is to have a download manager that is capable of downloading files on very unstable internet connections. All broken pipes should be resumed until the the total number of failures reach 30. In this case, the download status is changed to pause.

Please note that in the log window, only items marked in red are actual errors. The yellow one are warnings.

> don't seem to have an issue other than iweb.

I followed your instruction to get a large file from iweb and had no issue. That might be because I was on a really fast internet connection.

**Comment by wolfland** [comment]
Here is last nights testing results:

What I did..

2 Downloads with Turbo 0.2.8 to ensure I could reproduce the original 
problem. The first
download passed the second failed. This time the Turbo info window did 
not show the
"timeout" problem so I think we may have 2 different failure modes. 
Unfortunately I did
not capture any data with the Turbo 0.2.8 runs.

Installed Turbo 0.2.9-beta2 and ran 4 downloads of the same files. 2 
passed, 2 failed.

The 2 failures were not for the "timeout" failure but something different:

Actual downloadable URL is 
http://iweb.dl.sourceforge.net/project/luninuxos/16/LuninuXOS-16.04-Desktop-i386.iso

3:39:55 AM

Server seems to be done

3:41:16 AM

Server seems to be done

3:41:46 AM

Requesting a new thread

For some reason the Server seems to be done when it is not and it is 
clear that you are
requesting a new thread but nothing happens.

I wonder if there is a bug in the underlying subsystem(Firefox?) that 
causes the thread creation
request to fail.

In the past I have run the "Beta" version of DownThemAll as it seemed to 
behave better under
certain conditions than the released version. Unfortunately Firefox 
forces use of add-ons from
their servers only so a beta cannot be used unless it is on their server 
so I am forced to use
the released version.

I am on 4Mb/s DSL which is reasonably fast and I usually get 400Kb/s 
minimum to 4Mb/s maximum
depending on conditions as well as how fast the server/mirror is.

I want to do more downloads on the machine that is connected directly to 
my ethernet switch and
DSL router as I did not see any failures last time I tried there. I only 
ran 2 downloads and want to
run at least 4 to rule out a problem with my wireless.

Anyway I think we are making progress as I did not see the timeout 
problem with 0.2.9-beta2.

I will attach the Turbo infowindow files as well as the one console I 
was able to capture and
2 screenshots.

Thanks again for your hard work,
Brett

On 05/26/2016 01:27 AM, InBasic wrote:

> ```
> I am still eliminating variables at my end
> ```
> 
> There is no need. The whole idea behind TDM is to have a download 
> manager that is capable of downloading files on very unstable internet 
> connections. All broken pipes should be resumed until the the total 
> number of failures reach 30. In this case, the download status is 
> changed to pause.
> 
> Please note that in the log window, only items marked in red are 
> actual errors. The yellow one are warnings.
> 
> ```
> don't seem to have an issue other than iweb.
> ```
> 
> I followed your instruction to get a large file from iweb and had no 
> issue. That might be because I was on a really fast internet connection.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub 
> https://github.com/inbasic/turbo-download-manager/issues/50#issuecomment-221780922

'mozHidden' and 'mozVisibilityState' are deprecated.  Please use the unprefixed 'hidden' and 'visibilityState' instead. pubads_impl_88.js:54:131
The page was reloaded, because the character encoding declaration of the HTML document was not found when prescanning the first 1024 bytes of the file. The encoding declaration needs to be moved to be within the first 1024 bytes of the file. index.html:49:0
======== /41014381/Sourceforge/SF_ProjectFiles_728x90_A viewability met   ================== sf.head.js:8:1157
 [  0.003s] [studio.sdk]  Enabler.js:112:144
 [  0.064s] [studio.sdk] Enabler received un-whitelisted message. Enabler.js:112:144
Message from worker sca.js:13:335
Object { src: "36a050", workerStart: 1464244015678, count: 0, result: "1c7909b02df8a344e03a994408556a7dc35…", workerEnd: 1464244015681 } sca.js:13:370
Message from worker sca.js:13:335
Object { target: "736", src: "36a050", cap: 1000000, workerStart: 1464244015826, count: 1140, result: "73608587417cdac80d37df38ad1f31c69bd…", workerEnd: 1464244015901 } sca.js:13:370
Message from worker sca.js:13:335
Object { target: "68c", src: "36a050", cap: 1000000, workerStart: 1464244015949, count: 1614, result: "68c9ab4e14771dec25e52d03db5121b2323…", workerEnd: 1464244016079 } sca.js:13:370
Message from worker sca.js:13:335
Object { target: "8c5", src: "36a050", cap: 1000000, workerStart: 1464244015950, count: 1574, result: "8c5c90418616f4a3d68e30259e7fe92fb43…", workerEnd: 1464244016080 } sca.js:13:370
 [  0.843s] [studio.sdk] Enabler received un-whitelisted message. Enabler.js:112:144
'mozHidden' and 'mozVisibilityState' are deprecated.  Please use the unprefixed 'hidden' and 'visibilityState' instead. lidar.js:27:136
Message from worker sca.js:13:335
Object { target: "368", src: "36a050", cap: 1000000, workerStart: 1464244015951, count: 10423, result: "36800f45eb9a0a70d3039286a2d233b168e…", workerEnd: 1464244016527 } sca.js:13:370
 [  1.040s] [studio.sdk] Using default ad parameters in test environment. Simulating local events. Enabler.js:112:144
Total time: 13s main.js:1:101
Message from worker sca.js:13:335
Object { target: "c5e", src: "36a050", cap: 1000000, workerStart: 1464244015950, count: 23061, result: "c5e78e6d043a2b3120d600cc726d5f44f98…", workerEnd: 1464244017064 } sca.js:13:370
======== /41014381/Sourceforge/SF_ProjectFiles_300x250_A viewability met   ================== sf.head.js:8:1157
'mozHidden' and 'mozVisibilityState' are deprecated.  Please use the unprefixed 'hidden' and 'visibilityState' instead. pubads_impl_88.js:54:131
'mozHidden' and 'mozVisibilityState' are deprecated.  Please use the unprefixed 'hidden' and 'visibilityState' instead. container.html:17:1040
An unbalanced tree was written using document.write() causing data from the network to be reparsed. For more information https://developer.mozilla.org/en/Optimizing_Your_Pages_for_Speculative_Parsing container.html:17:0
======== /41014381/Sourceforge/SF_ProjectFiles_300x250_A viewability met   ================== sf.head.js:8:1157
======== /41014381/Sourceforge/SF_ProjectFiles_728x90_A viewability met   ================== sf.head.js:8:1157
TypeError: SF.Ads.showContentForAdblock is not a function
 download:837:1
'mozHidden' and 'mozVisibilityState' are deprecated.  Please use the unprefixed 'hidden' and 'visibilityState' instead. pubads_impl_88.js:54:131
syntax error download:1:1
Exit TRCRBox.loadScriptCallback(retry=1): no items in response - thumbnails-b impl.210-30-RELEASE.js:2:718
An unbalanced tree was written using document.write() causing data from the network to be reparsed. For more information https://developer.mozilla.org/en/Optimizing_Your_Pages_for_Speculative_Parsing container.html:17:0
An unbalanced tree was written using document.write() causing data from the network to be reparsed. For more information https://developer.mozilla.org/en/Optimizing_Your_Pages_for_Speculative_Parsing container.html:17:0
======== /41014381/Sourceforge/SF_Temp5_728x90_A viewability met   ================== sf.head.js:8:1157
======== /41014381/Sourceforge/SF_Temp5_multi_A viewability met   ================== sf.head.js:8:1157
An unbalanced tree was written using document.write() causing data from the network to be reparsed. For more information https://developer.mozilla.org/en/Optimizing_Your_Pages_for_Speculative_Parsing container.html:17:0
'mozHidden' and 'mozVisibilityState' are deprecated.  Please use the unprefixed 'hidden' and 'visibilityState' instead. container.html:19:5562
An unbalanced tree was written using document.write() causing data from the network to be reparsed. For more information https://developer.mozilla.org/en/Optimizing_Your_Pages_for_Speculative_Parsing container.html:17:0
======== /41014381/Sourceforge/SF_Temp5_300x250_A viewability met   ================== sf.head.js:8:1157
Array [ "Setting up post-download redirector…" ] sf.dwnld.js:2:61691
itdmanager:application/octet-stream sandbox.js:340
Use of getPreventDefault() is deprecated.  Use defaultPrevented instead. index.html
'mozHidden' and 'mozVisibilityState' are deprecated.  Please use the unprefixed 'hidden' and 'visibilityState' instead. pubads_impl_88.js:54:131
syntax error postdownload:1:1
TypeError: SF.Ads.showContentForAdblock is not a function
 postdownload:846:1
Setting up post-download redirector with a delay of 60000 to /projects/luninuxos/?source=typ_redirect. postdownload:918:5
Exit TRCRBox.loadScriptCallback(retry=1): no items in response - thumbnails-b impl.210-30-RELEASE.js:2:718
syntax error postdownload:1:1
An unbalanced tree was written using document.write() causing data from the network to be reparsed. For more information https://developer.mozilla.org/en/Optimizing_Your_Pages_for_Speculative_Parsing container.html:17:0
An unbalanced tree was written using document.write() causing data from the network to be reparsed. For more information https://developer.mozilla.org/en/Optimizing_Your_Pages_for_Speculative_Parsing container.html:17:0
Would redirect to URL: /projects/luninuxos/?source=typ_redirect postdownload:922:9

2:27:21 AM  

Downloading http://iweb.dl.sourceforge.net/project/luninuxos/16/LuninuXOS-16.04-Desktop-i386.iso

2:27:21 AM  

Referrer page is https://sourceforge.net/projects/luninuxos/files/16/LuninuXOS-16.04-Desktop-i386.iso/download

2:27:21 AM  

Download status is changed to download

2:27:21 AM  

Allocating space ...

2:27:21 AM  

File mime type is application/octet-stream

2:27:21 AM  

File encoding is null

2:27:21 AM  

Server multi-threading support status is true

2:27:21 AM  

File length in bytes is 2091821056

2:27:21 AM  

Actual downloadable URL is http://iweb.dl.sourceforge.net/project/luninuxos/16/LuninuXOS-16.04-Desktop-i386.iso

3:39:55 AM  

Server seems to be done

3:41:16 AM  

Server seems to be done

3:41:46 AM  

Requesting a new thread

Downloading http://iweb.dl.sourceforge.net/project/luninuxos/16/LuninuXOS-16.04-Desktop-amd64.iso

10:15:03 PM 

Referrer page is https://sourceforge.net/projects/luninuxos/files/16/LuninuXOS-16.04-Desktop-amd64.iso/download

10:15:04 PM 

Download status is changed to download

10:15:04 PM 

Allocating space ...

10:15:04 PM 

File mime type is application/octet-stream

10:15:04 PM 

File encoding is null

10:15:04 PM 

Server multi-threading support status is true

10:15:04 PM 

File length in bytes is 1933889536

10:15:04 PM 

Actual downloadable URL is http://iweb.dl.sourceforge.net/project/luninuxos/16/LuninuXOS-16.04-Desktop-amd64.iso

10:19:52 PM 

Server seems to be done

11:23:50 PM 

Server seems to be done

11:24:20 PM 

Requesting a new thread

**Comment by inbasic** [comment]
Thanks for the detailed report. Would you please give the latest dev version a try and let me know if the issue is resolved or not?

https://github.com/inbasic/turbo-download-manager/blob/master/builds/packed/firefox.xpi?raw=true

Open about:debugging and point the browse button to the XPI file.

In this build a bug in the Firefox implementation of the fetch API is fixed which I hope fixes this issue as well.

The new log should throw timeout error and hence restart the thread:

```
11:25:49 AM    Server seems to be done 
11:25:56 AM fetch error[3]; "timeout"
11:26:05 AM Server seems to be done 
11:26:14 AM fetch error[3]; "timeout"
```

**Comment by wolfland** [comment]
I tested the new version today and it seems to be working fine.

All of the downloads completed without intervention execept 1
which did not start(just kept gettings retries). I think this may
be due to the server that got selected as the download worked
fine when I tried it again and another server was selected.

I was not able to test against the iweb server as it was not in the
pool today but I don't think it matters as I have seen failures on
other servers since our last email.

I will attach infowindow contents for both the completed as well
as the 1 failed download.

Let me know if there is anything more I can do and I am always
willing to help test if you need it in the future.

Oh yeah almost forgot.. I did all my testing using Firefox 46 which
is the released version for Ubuntu 16.04. I also have Firefox 48
installed (e.g. Nightly) which has multithreading enabled so I can
also test using it if needed.

Thanks,
Brett

On 06/06/2016 02:58 AM, InBasic wrote:

> Thanks for the detailed report. Would you please give the latest dev 
> version a try and let me know if the issue is resolved or not?
> 
> https://github.com/inbasic/turbo-download-manager/blob/master/builds/packed/firefox.xpi?raw=true
> 
> Open about:debugging and point the browse button to the XPI file.
> 
> In this build a bug in the Firefox implementation of the fetch API is 
> fixed which I hope fixes this issue as well.
> 
> The new log should throw timeout error and hence restart the thread:
> 
> |11:25:49 AM Server seems to be done 11:25:56 AM fetch error[3]; 
> "timeout" 11:26:05 AM Server seems to be done 11:26:14 AM fetch 
> error[3]; "timeout" |
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub 
> https://github.com/inbasic/turbo-download-manager/issues/50#issuecomment-223882019, 
> or mute the thread 
> https://github.com/notifications/unsubscribe/AIjIgEAMYW9oAlENEdZu6Tk4L6kQ4bu-ks5qI8UUgaJpZM4Ij5SC.

Downloading http://pilotfiber.dl.sourceforge.net/project/manjarolinux/release/16.06/kde/manjaro-kde-16.06-x8664.iso

11:28:25 AM 

Referrer page is https://sourceforge.net/projects/manjarolinux/files/release/16.06/kde/manjaro-kde-16.06-x8664.iso/download?usemirror=pilotfiber

11:28:40 AM 

Download status is changed to download

11:28:40 AM 

Allocating space ...

11:28:40 AM 

File mime type is application/octet-stream

11:28:40 AM 

File encoding is null

11:28:40 AM 

Server multi-threading support status is true

11:28:40 AM 

File length in bytes is 2026078208

11:28:40 AM 

Actual downloadable URL is http://pilotfiber.dl.sourceforge.net/project/manjarolinux/release/16.06/kde/manjaro-kde-16.06-x8664.iso

11:34:00 AM 

Server seems to be done

12:04:00 PM 

Server seems to be done

12:06:02 PM 

Server seems to be done

12:20:02 PM 

Server seems to be done

12:24:39 PM 

fetch error[3]; "timeout"

12:36:00 PM 

Server seems to be done

12:38:00 PM 

Server seems to be done

12:40:01 PM 

Server seems to be done

12:41:20 PM 

Download status is changed to done

12:41:20 PM 

MD5 checksum is MD5 calculation is skipped

Downloading http://pilotfiber.dl.sourceforge.net/project/manjarolinux/release/16.06/kde/manjaro-kde-16.06-i686.iso

12:50:53 PM 

Referrer page is https://sourceforge.net/projects/manjarolinux/files/release/16.06/kde/manjaro-kde-16.06-i686.iso/download

12:51:01 PM 

Download status is changed to download

12:51:01 PM 

Allocating space ...

12:51:01 PM 

File mime type is application/octet-stream

12:51:01 PM 

File encoding is null

12:51:01 PM 

Server multi-threading support status is true

12:51:01 PM 

File length in bytes is 1831075840

12:51:01 PM 

Actual downloadable URL is http://pilotfiber.dl.sourceforge.net/project/manjarolinux/release/16.06/kde/manjaro-kde-16.06-i686.iso

12:51:31 PM 

fetch error[3]; "timeout"

12:58:02 PM 

Server seems to be done

1:06:01 PM  

Server seems to be done

1:18:01 PM  

Server seems to be done

1:40:02 PM  

Server seems to be done

1:46:21 PM  

fetch error[3]; "timeout"

1:50:02 PM  

Server seems to be done

1:54:02 PM  

Server seems to be done

1:56:38 PM  

Download status is changed to done

1:56:38 PM  

MD5 checksum is MD5 calculation is skipped

2:55:49 PM  

Downloading http://freefr.dl.sourceforge.net/project/manjarolinux/community/Cinnamon/16.06/x8664/manjaro-cinnamon-16.06-x8664.iso

2:55:49 PM  

Referrer page is https://sourceforge.net/projects/manjarolinux/files/community/Cinnamon/16.06/x8664/manjaro-cinnamon-16.06-x8664.iso/download

2:55:50 PM  

Download status is changed to download

2:55:50 PM  

Allocating space ...

2:55:50 PM  

File mime type is application/octet-stream

2:55:50 PM  

File encoding is null

2:55:50 PM  

Server multi-threading support status is true

2:55:50 PM  

File length in bytes is 1549615104

2:55:50 PM  

Actual downloadable URL is http://freefr.dl.sourceforge.net/project/manjarolinux/community/Cinnamon/16.06/x8664/manjaro-cinnamon-16.06-x8664.iso

2:55:50 PM  

fetch error[5]; "expected 206 but got 200"

2:55:51 PM  

fetch error[5]; "expected 206 but got 200"

2:55:51 PM  

fetch error[5]; "expected 206 but got 200"

2:55:52 PM  

fetch error[5]; "expected 206 but got 200"

2:55:56 PM  

fetch error[5]; "expected 206 but got 200"

2:55:56 PM  

fetch error[5]; "expected 206 but got 200"

2:55:57 PM  

fetch error[5]; "expected 206 but got 200"

2:56:00 PM  

fetch error[5]; "expected 206 but got 200"

2:56:01 PM  

fetch error[5]; "expected 206 but got 200"

2:56:01 PM  

fetch error[5]; "expected 206 but got 200"

2:56:02 PM  

fetch error[5]; "expected 206 but got 200"

2:56:03 PM  

fetch error[5]; "expected 206 but got 200"

2:56:04 PM  

fetch error[5]; "expected 206 but got 200"

2:56:05 PM  

fetch error[5]; "expected 206 but got 200"

2:56:05 PM  

fetch error[5]; "expected 206 but got 200"

2:56:06 PM  

fetch error[5]; "expected 206 but got 200"

2:56:07 PM  

fetch error[5]; "expected 206 but got 200"

2:56:07 PM  

fetch error[5]; "expected 206 but got 200"

2:56:09 PM  

fetch error[5]; "expected 206 but got 200"

2:56:10 PM  

fetch error[5]; "expected 206 but got 200"

2:56:10 PM  

fetch error[5]; "expected 206 but got 200"

2:56:11 PM  

fetch error[5]; "expected 206 but got 200"

2:56:12 PM  

fetch error[5]; "expected 206 but got 200"

2:56:13 PM  

fetch error[5]; "expected 206 but got 200"

2:56:14 PM  

fetch error[5]; "expected 206 but got 200"

2:56:15 PM  

fetch error[5]; "expected 206 but got 200"

2:56:15 PM  

Download status is changed to pause

Downloading http://pilotfiber.dl.sourceforge.net/project/manjarolinux/community/Cinnamon/16.06/x8664/manjaro-cinnamon-16.06-x8664.iso

2:57:43 PM  

Referrer page is https://sourceforge.net/projects/manjarolinux/files/community/Cinnamon/16.06/x8664/manjaro-cinnamon-16.06-x8664.iso/download

2:57:59 PM  

Download status is changed to download

2:57:59 PM  

Allocating space ...

2:57:59 PM  

File mime type is application/octet-stream

2:57:59 PM  

File encoding is null

2:57:59 PM  

Server multi-threading support status is true

2:57:59 PM  

File length in bytes is 1549615104

2:57:59 PM  

Actual downloadable URL is http://pilotfiber.dl.sourceforge.net/project/manjarolinux/community/Cinnamon/16.06/x8664/manjaro-cinnamon-16.06-x8664.iso

2:58:29 PM  

fetch error[3]; "timeout"

2:58:29 PM  

fetch error[3]; "timeout"

2:59:00 PM  

fetch error[3]; "timeout"

2:59:30 PM  

fetch error[3]; "timeout"

3:00:01 PM  

fetch error[3]; "timeout"

3:08:01 PM  

Server seems to be done

3:10:01 PM  

Server seems to be done

3:16:02 PM  

Server seems to be done

3:51:51 PM  

Server seems to be done

3:53:55 PM  

Download status is changed to done

3:53:55 PM  

MD5 checksum is MD5 calculation is skipped

Downloading http://pilotfiber.dl.sourceforge.net/project/c4clubunturespin/c4clrs64.iso

4:17:39 PM  

Referrer page is https://sourceforge.net/projects/c4clubunturespin/files/c4clrs64.iso/download

4:18:11 PM  

Download status is changed to download

4:18:11 PM  

Allocating space ...

4:18:11 PM  

File mime type is application/octet-stream

4:18:11 PM  

File encoding is null

4:18:11 PM  

Server multi-threading support status is true

4:18:11 PM  

File length in bytes is 4293918720

4:18:11 PM  

Actual downloadable URL is http://pilotfiber.dl.sourceforge.net/project/c4clubunturespin/c4clrs64.iso

4:18:41 PM  

fetch error[3]; "timeout"

4:18:42 PM  

fetch error[3]; "timeout"

4:18:42 PM  

fetch error[3]; "timeout"

4:24:01 PM  

Server seems to be done

4:28:01 PM  

Server seems to be done

4:38:02 PM  

Server seems to be done

4:46:00 PM  

Server seems to be done

4:52:41 PM  

Server seems to be done

4:52:47 PM  

fetch error[3]; "timeout"

4:58:02 PM  

Server seems to be done

5:04:02 PM  

Server seems to be done

5:10:01 PM  

Server seems to be done

5:14:02 PM  

Server seems to be done

5:16:01 PM  

Server seems to be done

5:28:01 PM  

Server seems to be done

5:29:00 PM  

fetch error[3]; "timeout"

5:36:01 PM  

Server seems to be done

5:38:00 PM  

Server seems to be done

5:41:50 PM  

Server seems to be done

5:50:02 PM  

Server seems to be done

5:56:02 PM  

Server seems to be done

6:06:02 PM  

Server seems to be done

6:10:01 PM  

Server seems to be done

6:16:02 PM  

Server seems to be done

6:18:02 PM  

Server seems to be done

6:34:00 PM  

Server seems to be done

6:50:01 PM  

Server seems to be done

6:53:11 PM  

Download status is changed to done

6:53:11 PM  

MD5 checksum is MD5 calculation is skipped

**Comment by inbasic** [comment]
The one that failed is definitely due to a server issue (`fetch error[5]; "expected 206 but got 200"`), so no worries there. Seems like the main issue has been resolved.

Thanks for your contribution!
